sdk/keyvault/azure-security-keyvault-keys/tsp-location.yaml

```
@@ -0,0 +1,5 @@
+directory: specification/keyvault/Security.KeyVault.Keys
+commit: bc18a6a035dbab575177a5548c936621e47d1124
+repo: Azure/azure-rest-api-specs
+additionalDirectories: 
+- specification/keyvault/Security.KeyVault.Common/
```

Can we put EOL@EOF here?
```suggestion
- specification/keyvault/Security.KeyVault.Common/

```

---

done